spec/factories/workflows.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :workflow, class: Sipity::Workflow do
-    name 'generic_work'
+    sequence(:name) { |n| "generic_work-#{n}" }
```

Comment:
Yes, this change is needed. This is because now that we are creating sipity entities, these workflows need to exist and not having this will throw the `UNIQUE constraint failed` errors in the DB